superset/assets/visualizations/filter_box.jsx

```diff
@@ -72,26 +72,38 @@ class FilterBox extends React.Component {
       maxes[filter] = d3.max(data, function (d) {
         return d.metric;
       });
+      const val = this.state.selectedValues[filter];
+      if (Array.isArray(val)) {
+        val.forEach(function (v) {
+          const exists = data.find(function (e) {
```

Comment:
could also simplify this line to: `const exists = data.find((e) => e.id === v);`